components/layout/multicol.rs

```diff
@@ -74,9 +74,9 @@ impl Flow for MulticolFlow {
         &self.block_flow
     }
 
-    fn as_mut_multicol(&mut self) -> &mut MulticolFlow {
+    /*fn as_mut_multicol(&mut self) -> &mut MulticolFlow {
         self
-    }
+    }*/
```

Comment:
What's the deal with this?

Reply:
@jdm thanks for pointing this out. I initially always comment out the code because my tool outputs file_name.rs:line:col information and removing would invalidate that info. Later on I then remove it using git diff but seems I've missed this one. I've removed it.